src/main/java/com/prgrms/springJpa/domain/customer/Customer.java

```diff
@@ -13,7 +14,7 @@
 @Table
```

Comment:
```suggestion
@Table
```
`@Table` 은 사용하지 않고 있는 것 같습니다!

Reply:
해당 테이블을 이번 미션에서 사용하지 않는다는 말씀이시죠??
저번 미션때 사용했던 건데 머지를 잘못해서 이번 미션에서도 보이네요 😂

Reply:
Entity만 사용해도 JPA에서 테이블로 만들어주어서 저는 Entity만 사용을 했어서요! Table을 사용하는 이유가 있나요 😃 ❓